mmtrack/datasets/mot_challenge_dataset.py

```diff
@@ -374,12 +373,35 @@ def evaluate(self,
                 else:
                     gt_file = osp.join(self.img_prefix, f'{name}/gt/gt.txt')
                 res_file = osp.join(resfiles['track'], f'{name}.txt')
-                gt = mm.io.loadtxt(gt_file)
+                # copy gt file from ceph to local temporary directory
+                # no 'self.img_prefix'
+                gt_dir_path = osp.join(local_dir.name, name, 'gt')
+                os.makedirs(gt_dir_path)
+                gt_file_tmp = osp.join(
+                    local_dir.name,
+                    gt_file.replace(gt_file.split(name)[0], ''))
+
+                f = open(gt_file_tmp, 'wb')
+                gt_content = self.file_client.get(gt_file)
+                if hasattr(gt_content, 'tobytes'):
+                    gt_content = gt_content.tobytes()
+                f.write(gt_content)
+                f.close()
+                # copy sequence file from ceph to local temporary directory
+                seqinfo_path = osp.join(local_dir.name, name, 'seqinfo.ini')
```

Comment:
The `seqinfo.ini` may not exist.
Therefore, we need to copy `seqinfo.ini` only when it exist.

Reply:
But this file maybe necessary, because it will be used again on line **442**.
`dataset = [trackeval.datasets.MotChallenge2DBox(dataset_config)]`
If we don't throw errors here, we will still throw errors later.